tests/static/e2e/routes.ts

```diff
@@ -0,0 +1,9 @@
+export const orgSlug = "default"
+
+export enum Routes {
```

Comment:
Would be nice if we could import these directly from the Vue router. I was unable to import the routes because of an issue with this line and others upon import (can't use this outside module, or similar error):

```
const authProviderSlug =
  import.meta.env.VITE_DISPATCH_AUTHENTICATION_PROVIDER_SLUG || "dispatch-auth-provider-basic"
```